pages/content/posts/complete-interop-features/index.md

```diff
@@ -0,0 +1,69 @@
+
+The third step will give us a complete reference for the Forms APIs. It will be synchronized with this year's Interop initiative, where all three engines will be interoperable.
+
+## Current progress
+
+During the first Quarter of 2023, we focused on the first point, documenting all interoperable APIs unrelated to HTML and SVG (See our [tracking project](https://github.com/openwebdocs/project/issues/152)). We added more than 100 pages to MDN, documenting missing modern features like [`CSSLayerBlockRule`](https://developer.mozilla.org/en-US/docs/Web/API/CSSLayerBlockRule) and [`CSSLayerStatementRule`](https://developer.mozilla.org/en-US/docs/Web/API/CSSLayerStatementRule), or old features writers forgot over the years, like [`XMLHttpRequestUpload`](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequestUpload).
+
+Already we can detect missed interoperable, unrelated to HTML and SVG DOMs, features that become interactive: for example, we added the [`CSSFontFeaturesValuesRule`](https://developer.mozilla.org/en-US/docs/Web/API/CSSFontFeatureValuesRule) that became interoperable in Q1.
+
+In this Quarter, we will document HTML APIs not part of the HTML DOM, interfaces like `ErrorEvent`, `HTMLAllCollection`, or even `HTMLOptionsCollection`. You can follow our progress in our [tracking GitHub issue](https://github.com/openwebdocs/project/issues/153) In the future, writers will link numerous times to these documents. New pages will not have _red links_, links to non-written pages, improving the quality of our new docs. We also plan to improve the documentation for the _Constraint Validation API_ this Quarter.
+
+Then, in Q3, we plan to tackle the last point, the missing `HTML*Element` features related to forms. Overall we plan to add 350 pages this year, about 30% of the overall gaps (for interoperable features).
+
+## And all the rest?
+
+About 700 more interoperable properties and methods will not be documented after this first project. Even if we do not have a plan, we have a few ideas to tackle the remaining gaps. For SVG, we would like to take a similar approach. First, we would like to document fundamental interfaces, like the types: `SVGLength`, `SVGAnimatedLength`, … In the second phase, we would like to call the community for help. There are a lot of similar docs to write, like the different kinds of SVG filters. The pages will have the same structure, although the examples will differ. We will see what will happen, and hopefully, we can report back on progress here!
```

Comment:
```suggestion
After this interop documentation project is complete, almost 700 more interoperable properties and methods will still need to be documented. We are developing plans to tackle the remaining gaps. 

SVG documentation could benefit from a similar interop approach. In a first phase, we would like to document fundamental interfaces, like the types: `SVGLength`, `SVGAnimatedLength`. We would like to call the community for help in the second phase: there are a lot of similar docs to write, like the multitude of SVG filters. Pages with similar structures, though with different examples, are a great opportunity for community involvement. 

We will see what will happen, and hopefully, we can report back on progress here!
```

Reply:
I didn't want to say "first" project, since this is the second interop (or third) as interop 2022 was a project, and interop 2023 is too, though that one hasn't started so not sure if that is 2nd or 3rd